app/controllers/course-admin/tester-version.ts

```diff
@@ -0,0 +1,82 @@
+import Controller from '@ember/controller';
+import CourseTesterVersionModel from 'codecrafters-frontend/models/course-tester-version';
+import Store from '@ember-data/store';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class CourseAdminTesterVersionController extends Controller {
+  declare model: {
+    course: {
+      id: string;
+      definitionRepositoryFullName: string;
+      syncCourseDefinitionUpdates: () => Promise<void>;
+    };
+
+    testerVersion: CourseTesterVersionModel;
+  };
+
+  @service declare store: Store;
+
+  @tracked isActivating = false;
+  @tracked isDeprovisioningTestRunners = false;
+  @tracked shouldShowDeprovisioningNotice = false;
+
+  get activationConfirmationMessage() {
+    // @ts-ignore
```

Comment:
@libmartinito can we do `@ts-expect-error` instead, and add a reason for this? I don't see why this should fail - `testerVersion` _should_ have `course` defined on it no?